mantidproject/mantid
# Dev-docs: Writing an algorithm, custom ConvertToMD plugin

Adds these pages to the dev-docs:

* [Writing custom ConvertTo MD transformation](http://www.mantidproject.org/Writing%20custom%20ConvertTo%20MD%20transformation)
* [Writing an Algorithm](http://www.mantidproject.org/Writing%20an%20Algorithm)

**To test:**

Please read and make sure they still make sense since I've edited the ConvertToMD one quite a bit.
Also check links etc.

Refs #22082
